### ISSUE TITLE
#error   fatal error: toolkits/point_cloud/MapManager.hpp: 没有那个文件或目录

### ISSUE THREAD

**Issue by CHANdaFeng** [rsasaki0109/lidar_localizer#11]
Hello, I recently encountered this problem in the compilation process. Please tell me how to solve it？ Thank you～
In file included from /home/cxf/mkdir test_ws/ndt_mapping_ws/src/ndt_mapping-master/src/ndt_mapping.cpp:23:
/home/cxf/mkdir test_ws/ndt_mapping_ws/src/ndt_mapping-master/include/ndt_mapping.h:47:10: fatal error: toolkits/point_cloud/MapManager.hpp: 没有那个文件或目录
47 | #include <toolkits/point_cloud/MapManager.hpp>
| ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from /home/cxf/mkdir test_ws/ndt_mapping_ws/src/ndt_mapping-master/src/ndt_mapping_node.cpp:22:
/home/cxf/mkdir test_ws/ndt_mapping_ws/src/ndt_mapping-master/include/ndt_mapping.h:47:10: fatal error: toolkits/point_cloud/MapManager.hpp: 没有那个文件或目录
47 | #include <toolkits/point_cloud/MapManager.hpp>
| ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

**Comment by rsasaki0109** [comment]
I'm guessing you are not using my package because my package does not include toolkits on the line listed in the error section.
Which package are you using? I think it would be a good idea to ask the person who is publishing that package.
https://github.com/rsasaki0109/lidar_localizer/blob/main/include/ndt_mapping.h#L47

**Comment by rsasaki0109** [comment]
Could be this same problem.
https://github.com/rsasaki0109/lidar_localizer/issues/6

**Comment by CHANdaFeng** [comment]
Indeed, as you said, I compiled an author who referenced your function package, and now it has been compiled successfully, thank you very much!
